pkg/agent/flowexporter/connections/connections.go

```diff
@@ -0,0 +1,116 @@
+package connections
+
+import (
+	"sync"
+	"time"
+
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+	"github.com/vmware-tanzu/antrea/pkg/agent/interfacestore"
+	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
+)
+
+var _ ConnectionStore = new(connectionStore)
+
+type ConnectionStore interface {
+	Run(stopCh <-chan struct{})
+}
+
+type connectionStore struct {
+	connections map[flowexporter.ConnectionKey]flowexporter.Connection // Add 5-tuple as string array
+	connTrack   ConnTrack
+	ifaceStore  interfacestore.InterfaceStore
+	mutex       sync.Mutex
+}
+
+func NewConnectionStore(connTrack ConnTrack, ifaceStore interfacestore.InterfaceStore) *connectionStore {
+	return &connectionStore{
+		connections: make(map[flowexporter.ConnectionKey]flowexporter.Connection),
+		connTrack:   connTrack,
+		ifaceStore:  ifaceStore,
+	}
+}
+
+// Run polls the connTrack module periodically to get connections. These connections are used
+// to build connection store.
+func (cs *connectionStore) Run(stopCh <-chan struct{}) {
+	klog.Infof("Starting conntrack polling")
+	for {
+		select {
+		case <-stopCh:
+			break
+		case <-time.After(flowexporter.PollInterval):
```

Comment:
`time.After` will create a new Timer every time it's invoked, I think creating a Ticker is more suitable here.
```
	ticker := time.NewTicker(flowexporter.PollInterval)
	defer ticker.Stop() 
	for {
		select {
		case <-stopCh:
			break
		case <-ticker.C:
                         ...

```

Reply:
Done. Thanks for the suggestion.